content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -407,38 +407,40 @@ The access modes are:
 * ReadWriteOnce -- the volume can be mounted as read-write by a single node
 * ReadOnlyMany -- the volume can be mounted read-only by many nodes
 * ReadWriteMany -- the volume can be mounted as read-write by many nodes
+* ReadWriteOncePod -- the volume can be mounted as read-write by a single pod {{< feature-state for_k8s_version="v1.22" state="alpha" >}}
```

Comment:
I am not sure the feature-state will show up correctly. You may need to make it a new paragraph.